app/templates/buyers/edit_brief_question.html

```diff
@@ -14,7 +14,7 @@
       },
       {
         "link": '/2/brief/%s/overview' % brief.id if brief.lot == 'digital-professionals' or brief.lot == 'training' else url_for(".view_brief_overview", framework_slug=brief.frameworkSlug, lot_slug=brief.lotSlug, brief_id=brief.id),
-        "label": "Brief Overview"
+        "label": "Opportunity Overview"
```

Comment:
There are occurrences in:
- brief_publish_confirmation.html
- brief_responses.html
- dashboard.html

Reply:
changed as suggested

Reply:
changes as suggested